tkg/tkgctl/helpers.go

```diff
@@ -214,6 +214,9 @@ func getFieldfromUnstructuredObject(obj unstructured.Unstructured, fields ...str
 }
 
 func setVSphereCredentialFromInputfile(legacyVarMap *map[string]string, clusterConfigFile, clusterName, namespace string) error {
+	if clusterName == "" {
```

Comment:
fwiw, this happened to me and indira the other day and we got confused :) if the name is empty you otherwise get an ugly lower level failure